model_analyzer/config/input/config_command_profile.py

```diff
@@ -297,6 +298,16 @@ def _fill_config(self):
                 description="Skips the generation of detailed summary reports and tables.",
             )
         )
+        self._add_config(
+            ConfigField(
+                "llm_mode",
+                flags=["--llm-mode"],
```

Comment:
I worry about adding a bunch of args in the future -- for other modes like embedding mode or video mode or something. I know that genai-perf was designed to be generic -- it is currently LLMs but the plan is to eventually expand it. I don't think they added an arg to specify the mode yet in genai-perf, but I feel like we should have a flag that takes in the mode instead of a bunch of boolean modes where only one can be set.

Reply:
I'm fine with that. I've changed the option to be `--model-type` with `generic` or `LLM` as options.